inlineplz/linters/eclint.py

```diff
@@ -3,8 +3,24 @@
 from __future__ import unicode_literals
 
 from inlineplz.parsers.base import ParserBase
+from inlineplz.decorators import linter
+import os.path
```

Comment:
[inline-plz]: `prospector: pylint: standard import "import os.path" should be placed before "from inlineplz.parsers.base import ParserBase" (wrong-import-order)`